frontend/layouts/annotation.vue

```diff
@@ -173,6 +177,9 @@ export default {
           path: this.localePath(`/projects/${this.$route.params.id}/`)
         })
       }
+      this.getCommentList({
```

Comment:
Should I fetch the comments in the comment section component?

Reply:
definitely.